packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -434,19 +434,40 @@
                 data-heading="{{:: 'pci_databases_general_information_service_integration_title' | translate }}"
             >
                 <oui-tile-definition
-                    data-term="{{:: 'pci_databases_general_information_kafka_mirror_maker_label' | translate }}"
+                    ng-if="$ctrl.database.engine === 'kafka'"
+                    data-term="{{::'pci_databases_general_information_kafka_mirror_maker_label' | translate }}"
                 >
                     <oui-tile-description>
                         <p
-                            data-ng-if="$ctrl.serviceIntegration.length === 0"
-                            data-translate="pci_databases_general_information_kafka_mirror_maker_none"
+                            data-ng-if="$ctrl.serviceIntegration.kafkaMirrorMaker.length === 0"
+                            data-translate="pci_databases_general_information_integration_none"
                         ></p>
                         <p
-                            data-ng-repeat="integration in $ctrl.serviceIntegration track by $index"
+                            data-ng-repeat="integration in $ctrl.serviceIntegration.kafkaMirrorMaker track by $index"
                         >
                             <span
                                 class="oui-badge_success oui-badge"
-                                data-ng-bind="integration.serviceName"
+                                data-ng-bind="integration.destinationServiceName"
+                            >
+                            </span>
+                        </p>
+                    </oui-tile-description>
+                </oui-tile-definition>
+                <oui-tile-definition
+                    ng-if="$ctrl.database.engine === 'm3db'"
```

Comment:
Can use constant here already defined in `DATABASE_TYPES` ?

Reply:
Done here: https://github.com/ovh/manager/pull/6426/commits/1b3b90052daed532663563561011f85f4811a532